.github/workflows/README.md

```diff
@@ -0,0 +1,23 @@
+# Github Actions Workflows
+
+## Benchmarking Performance Tests
+`benchmarks.yml` contains a workflow to check for any performance regressions or
+improvements in benchmark tests. 
+
+It uses
+[github-action-benchmark](https://github.com/benchmark-action/github-action-benchmark)
+to check performance diffs between a PR and the `master` branch, comments if it
+there is a regression past a certain threshold (default: `200%`), and generates
+a performance diff JSON between consecutive commits in the `master` branch in
+the `gh-pages` branch (the JSON is visualized into a graph that can be seen at:
+https://algorand.github.io/go-algorand/dev/bench/).
+
+### Adding benchmark tests
+Currently, the workflow runs the `BenchmarkUintMath` in the `Run benchmark`
```

Comment:
@algochoi Optional _suggestion_:  My bias is to minimize referencing material that's liable to change and instead document how to search for the content (possibly with a search link). 

While it seems unlikely today, it's possible `BenchmarkUintMath` will be renamed or even removed.  To improve the chance of keeping docs current, I propose different verbiage.

The proposal also suggests highlighting that we _deliberately_ haven't established naming + grouping conventions.

If you prefer as is, feel welcomed to resolve. 

```suggestion
Add run steps or extend existing benchmark invocations to add a new benchmark.  Look for `-bench` to find existing benchmarks.

Since there's few benchmarks run by the workflow, there's _no_ formal grouping and/or naming conventions.
```